object.rb

```diff
@@ -172,3 +172,65 @@ def Float(arg, exception: true)
     Primitive.rb_f_float(arg, exception)
   end
 end
+
+class TrueClass
+  #
+  # call-seq:
+  #   true.to_s   ->  "true"
+  #
+  # The string representation of <code>true</code> is "true".
+  #
+  def to_s
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'rb_cTrueClass_to_s'
+  end
+
+  def inspect
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'rb_cTrueClass_to_s'
+  end
+  #
+  #  call-seq:
+  #     true & obj    -> true or false
+  #
+  #  And---Returns <code>false</code> if <i>obj</i> is
+  #  <code>nil</code> or <code>false</code>, <code>true</code> otherwise.
+  #
+  def &(obj)
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'RTEST(obj)?  Qtrue : Qfalse'
+  end
+
+  #
+  #  call-seq:
+  #     true | obj   -> true
+  #
+  #  Or---Returns <code>true</code>. As <i>obj</i> is an argument to
+  #  a method call, it is always evaluated; there is no short-circuit
+  #  evaluation in this case.
+  #
+  #     true |  puts("or")
+  #     true || puts("logical or")
+  #
+  #  <em>produces:</em>
+  #
+  #     or
+  #
+  def |(obj)
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'Qtrue'
```

Comment:
Was this faster than Ruby `true`?

Reply:
bit slow than `Primitive.cexpr! 'Qtrue'`